### DISCUSSION TITLE
Can we list schools tenant wise or not? If yes then how

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#455] [Documentation] [answered]
We want to populate the **school list according to the tenant**.
If we can do it then please help us do it
**Which API we should use for it? Please share the Curl with us.**
![image](https://user-images.githubusercontent.com/109504832/230296249-b53c5ac9-a402-4096-8e3a-be71d119d1c8.png)
[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by rajeevsathish** [comment]
@reshmi-nair can you please look into this

**Comment by vinukumar-vs** [comment]
@sachinb80 Verify this thread for uploading the data
https://github.com/orgs/Sunbird-lern/discussions/92

Verify the search API is returning the data for the specific selection. 

@Krishnaj20 @rajeevsathish @reshmi-nair

**Reply by Krishnaj20** [reply]
@reshmi-nair @rhwarrier - can you point to Lern documentation which can address this query please?

**Reply by reshmi-nair** [reply, marked as answer]
@vinukumar-vs @sachinb80 Please use Org upload API for this.
[API documentation](https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/api-documentation/backend-services)

Request format:
curl --location --request POST 'localhost:9000/v1/org/upload' \
--header 'Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryOqHU2a1MzAKBayFA' \
--header 'Authorization: Bearer ' \
--header 'x-authenticated-user-token:' \
--form 'organisation=@"/home/reshmi/daily notes/org_upload_test.csv"'

Each successfully created organisation is associated with root org of uploader (org admin)

**Reply by Krishnaj20** [reply]
@sachinb80 - hope this answers your query.
